### PR TITLE
Move CMS tables to namespace

### PR DESCRIPTION
**Pull request by laoneo**
Move CMS tables to namespace.

### CODE REVIEW

**Comment by brianteeman** [comment]
if the class name changes then shouldnt the @since version change as well. #askingforafriend

**Comment by laoneo** [comment]
I don't think so as the class stays the same and does the same, just the name changed.

**Comment by mbabker** [comment]
Personally I wouldn't change them either.  If anything a `@note As of x.y this class was renamed from <old_class>` annotation would be as far as I'd go.

**Comment by brianteeman** [comment]
Thanks - was just asking

**Comment by photodude** [comment]
strange that this is failing with
```php
Fatal error: Class 'Joomla\Cms\Component\JFactory' not found in libraries\src\Joomla\Cms\Component\ComponentHelper.php on line 91
```

**Comment by laoneo** [comment]
You need to have this one installed first #14996, it fixes some merge invalidity.

**Comment by mbabker** [comment]
This needs syncing again please.

**Comment by laoneo** [comment]
Thanks @wilsonge. When one namespacing pr got merged, then there will be always a conflict in the classmap file in the other open ones.

File: libraries/src/Joomla/Cms/Table/ContentHistory.php [hunk 1]
```diff
@@ -71,7 +72,7 @@ public function __construct($db)
 	public function store($updateNulls = false)
 	{
 		$this->set('character_count', strlen($this->get('version_data')));
-		$typeTable = JTable::getInstance('Contenttype');
+		$typeTable = new ContentType($this->getDbo());
```
Comment (wilsonge) [inline libraries/src/Joomla/Cms/Table/ContentHistory.php]: Can we revert these changes please. Calling the function directly does have some b/c impact (where people adding paths to JTable can override the JTable used). If we're going to do this we should do it in 4.0 and not the 3.8 branch

**Comment by laoneo** [comment]
So, fixed some minor things and changed to getInstance
